doc/deprecated.md

```diff
@@ -19,6 +19,7 @@ A list of deprecated features and the version in which they were deprecated foll
 -   Support to Node.js v10 in iotagent-node-lib 2.15.0 (finally removed in 2.16.0)
 -   Support to Node.js v12 in iotagent-node-lib 2.24.0 (finally removed in 2.25.0)
 -   Support to NGSI-LD v1.3 in iotagent-node-lib 2.25.0
+-   Support to `deviceRegistry.type=memory` value. 
```

Comment:
Add also another item for the groups statically defined by configuration

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1318/commits/8a79b4a12949dd378b26d71124961f87d6a0d0a9